tensorflow/core/kernels/tile_op_test.cc

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please write these unit tests in Python rather than C++.  It makes them much easier to read.  You can add them to the existing tile tests.